frontend/lib/norent/components/letter-counter.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { useContext } from "react";
+import { AppContext } from "../../app-context";
+import { NorentLogo } from "./logo";
+
+// Initial count of norent.org letter sent, based on tally from LA letter sender
+const NORENT_LETTERS_SENT_PRE_LAUNCH = 122;
+
+export const LetterCounter = () => {
+  const { session } = useContext(AppContext);
+
+  return (
+    <div className="hero jf-letter-counter">
+      <div className="hero-body">
+        <div className="container has-text-centered">
+          <p className="title is-spaced has-text-info">
+            {session.norentLettersSent
+              ? NORENT_LETTERS_SENT_PRE_LAUNCH + session.norentLettersSent
+              : NORENT_LETTERS_SENT_PRE_LAUNCH}
```

Comment:
Hmm, I think that this value is defined to always be an integer, so you should be good with always adding it to `NORENT_LETTERS_SENT_PRE_LAUNCH`?